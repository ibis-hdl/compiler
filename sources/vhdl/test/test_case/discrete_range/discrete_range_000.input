-- discrete_subtype_indication (with typemark)
-- syntacially equivalent to range with range_attribute_name
typemark_or_range_attribute_name
