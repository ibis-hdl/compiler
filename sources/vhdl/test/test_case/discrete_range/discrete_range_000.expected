(discrete_range<v>
  (subtype_indication
    (name<v>
      (identifier
        typemark_or_range_attribute_name
      )
    )
  )
)
